packages/client/lib/config.ts

```diff
@@ -266,6 +266,8 @@ export interface ConfigOptions {
    * The time after which synced state is downgraded to unsynced
    */
   syncedStateRemovalPeriod?: number
+
+  maxStorageRange?: bigint
```

Comment:
can we move it below maxAccountRange for some context clustering

Reply:
done